ang/civicase/CaseCard.js

```diff
@@ -15,12 +15,22 @@
   });
 
   module.controller('CivicaseCaseCardController', function ($scope, getActivityFeedUrl, DateHelper) {
+    $scope.data.selected = $scope.data.selected || false;
+
     $scope.activityFeedUrl = getActivityFeedUrl;
     $scope.formatDate = DateHelper.formatDate;
     $scope.otherCategories = _.map(_.filter(CRM.civicase.activityCategories, function (category) {
       return category.name !== 'task' && category.name !== 'communication';
     }), function (category) {
       return category.name;
     });
+
+    /**
+     * Update the checkbox the case card
+     */
+    $scope.toggleSelected = function () {
+      $scope.data.selected = !$scope.data.selected;
+      $scope.$emit('bulkSelectionCheckboxClicked', $scope.data);
```

Comment:
Please follow this structure for emit messages.
`<appName>::<componentName/eventCategory>::<eventName>`
`'civicase::bulk-actions::check-box-toggled', $scope.activity);`